Improve docs (#679)
input-output-hk/cardano-wallet: lib/jormungandr/README.md

# Jörmungandr backend

> Jörmungandr is a node implementation, written in rust, with the initial aim

Reviewer: ```suggestion
Jörmungandr is a node implementation, written in Rust, with the initial aim
```

Author: done

> to support the Ouroboros type of consensus protocol.

Reviewer: ```suggestion
of supporting the Ouroboros type of consensus protocol.
```

Author: done

See [link](https://input-output-hk.github.io/jormungandr/) to get more information.

Instructions how to install Jörmungandr are [here](https://github.com/input-output-hk/jormungandr).

> After the installation one should see the below :

Reviewer: ```suggestion
After the installation, one can follow the steps below:
```

Author: done

``` bash
$ jcli --version
jcli 0.3.3
$ jormungandr --version
jormungandr 0.3.3
```

# Testing

``` bash
$ stack test cardano-wallet-jormungandr:integration
$ stack test cardano-wallet-jormungandr:unit
```

> For manual testing try :

Reviewer: ```suggestion
For manual testing, try:
```

Author: done

``` bash
$ pwd
.../cardano-wallet
$ stack build
$ cardano-wallet-jormungandr launch --genesis-block lib/jormungandr/test/data/jormungandr/block0.bin --bft-leaders lib/jormungandr/test/data/jormungandr/secret.yaml
```

> Here we are taking genesis block and bft leaders configuration as defined in integration tests.

Reviewer: ```suggestion
Here we are using a genesis block and BTF leaders configuration as defined in the integration tests.
```

Author: done

> For more options invoke help:

Reviewer: ```suggestion
For more options, see the help text:
```

Author: done

``` bash
$ stack exec -- cardano-wallet-jormungandr launch --help
```

> If the cardano-wallet part talking with Jörmungandr is running, then in the separate console

Reviewer: ```suggestion
If the `cardano-wallet` server is running with a Jörmungandr backend, then in a separate console
```

Author: done

> call the api, for example:

Reviewer: ```suggestion
call the API. For example:
```

Author: done
